hypervisor/common/hypercall.c

```diff
@@ -805,7 +805,14 @@ int32_t hcall_gpa_to_hpa(struct acrn_vm *vm, uint16_t vmid, uint64_t param)
 {
 	int32_t ret = -1;
 	struct vm_gpa2hpa v_gpa2hpa;
-	struct acrn_vm *target_vm = get_vm_from_vmid(vmid);
```

Comment:
This solution is not good enough. get_vm_from_vmid should work fine, but the vm_id of SOS changes. This is same for other VMs too.